homeassistant/components/sharkiq/vacuum.py

```diff
@@ -63,6 +66,15 @@ async def async_setup_entry(
     )
     async_add_entities([SharkVacuumEntity(d, coordinator) for d in devices])
 
+    platform = entity_platform.async_get_current_platform()
+    platform.async_register_entity_service(
+        SERVICE_CLEAN_ROOM,
+        {
+            vol.Required(ATTR_ROOMS): list,
```

Comment:
I would recommend to use `cv.ensure_list` and `vol.Length(min=1)` similar to https://github.com/home-assistant/core/blob/e2d2ec88178a5b1902dde15d41eddeb1d92ae64f/homeassistant/components/citybikes/sensor.py#L82-L84